### PR TITLE
Add support of OpenSearch 2.0

### PR DESCRIPTION
**Pull request by willyborankin**
Initial support of `OpenSearch` 2.0.0
- IOUtils class removed
- Switched to JDK 11

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -46,7 +41,8 @@ testingConventions.enabled = true
 
 buildscript {
     ext {
-        opensearch_version = "1.3.2"
+        opensearch_version = "2.0.0-rc1"
```
Comment (reta) [inline build.gradle:44]: `2.0.0` please, it is already out, artifacts published :partying_face:

File: build.gradle [hunk 2]
```diff
@@ -46,12 +35,14 @@ testingConventions.enabled = true
...
     ext {
-        opensearch_version = "1.3.2"
+        opensearch_version = "2.0.0"
+        bc_version = "1.70"
     }
 
     repositories {
         mavenLocal()
-        maven { url "https://aws.oss.sonatype.org/content/repositories/snapshots" }
+        maven { url "https://plugins.gradle.org/m2/" }
```
Comment (reta) [inline build.gradle:44]: Why do you need these ones?

Comment (reta) [reply build.gradle:44]: gradlePluginPortal() should be enough

Comment (willyborankin) [reply build.gradle:44]: well it is part of `https://github.com/opensearch-project/opensearch-plugin-template-java` template

**Review by reta — APPROVED** [review]

**Pull request merged**